### ISSUE TITLE
Modifying content types / help with select parameter

### ISSUE THREAD

**Issue by sngordon** [CrossRef/rest-api-doc#288]
I've found that when I request results an alternative format (ris, bibtex) the date returned tends to be the published-online date (perhaps the earliest of the published-online & published-print). The authors I'm working with tend to cite the published-print date, so I'm looking for strategies to provide the print date instead.

Examples:
http://api.crossref.org/works/10.1007/s10980-007-9188-1
ris date = published-online = 2007/12/22 (published-print = 2008/2)

http://api.crossref.org/works/10.1139/cjfr-2014-0148
ris date = published-print = 2015/2 (no published-online data)

I've thought of two options so far:

1. Modify the content negotiation / parsing code, but I don't see this available anywhere (I've only found this general reference that it is maintained by the registration agencies: https://citation.crosscite.org/docs.html).

2. Make an additional request to the API specifically for the published-print date using the SELECT parameter, and use it to replace the date in the RIS-formatted record. However, I've been unable to formulate a functional request using a specific doi and the select parameter - could someone provide an example? Here are a few that don't work: 

http://api.crossref.org/10.1007/s10980-007-9188-1/works?select=title,published-print
http://api.crossref.org/works/10.1007/s10980-007-9188-1/?select=title,published-print

**Comment by kjw** [comment]
Yes - we're probably using the earliest available publication date in most representations. I see that it makes sense to use the print publication date, where available, for representations that are used for citation. Will make a change here.

**Comment by jenniferlin15** [comment]
Ticketed - GO-918

**Comment by jenniferlin15** [comment]
Publication dates are a quandary due to their numerous use cases. It's not an issue when a specific one is requested so much as when we have to select & apply a particular one in the results. Adding this as a larger discussion item for the team.
